Formula/gpgme.rb

```diff
@@ -36,9 +37,16 @@ class Gpgme < Formula
     sha256 "5de2f6bcb6b30642d0cbc3fbd86803c9460d732f44a526f44cedee8bb78d291a"
   end
 
+  def python3
+    "python3.11"
+  end
+
   def install
-    python = "python3.10"
-    ENV["PYTHON"] = python
+    ENV["PYTHON"] = python3
+    # HACK: Stop build from ignoring our PYTHON input. As python versions are
+    # hardcoded, the Arch Linux patch that changed 3.9 to 3.10 can't detect 3.11
+    inreplace "configure", /# Reset everything.*\n\s*unset PYTHON$/, ""
```

Comment:
Alternatively:
- We could keep our own copy of patch and update to 3.11. 
- Replace patch with inreplace like Fedora where they sed the configure script to replace "3.8" to current Python version. Ref: https://src.fedoraproject.org/rpms/gpgme/blob/rawhide/f/gpgme.spec#_128

Not sure status upstream on previous PR (https://dev.gnupg.org/D546) which won't be enough on Python 3.11.

Reply:
As https://dev.gnupg.org/D546 does not look to move forward at a big pace. Let's add your hack until upstream has sorted things out.